<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindow</class>
 <widget class="QWidget" name="RegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>New User Registration</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>451</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>User registration form</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="0">
     <widget class="QLabel" name="lEmailid">
      <property name="text">
       <string>Email ID</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lFirstName">
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="leEmailId"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="leFirstName"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="lPassRecQ">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Password recovery question</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lePasswordRecoveryQ"/>
    </item>
    <item row="10" column="2">
     <widget class="QPushButton" name="pbRegister">
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="lPassRecA">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Password recovery answer</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="leDob"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="lePasswordRecoveryA"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="lDob">
      <property name="text">
       <string>Date of birth DD/MM/YYYY    </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lePassword"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lPassword">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lusername">
      <property name="text">
       <string>Username For chat</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pbCheckUsername">
      <property name="text">
       <string>Proceed</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="leUsername"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lPasswordConfirm">
      <property name="text">
       <string>Confirm Password</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lePasswordConfirm"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="llastName">
      <property name="text">
       <string>Last Name</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="leLastName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
